test/functional-portable/testUtil.go

```diff
@@ -0,0 +1,332 @@
+/*
```

Comment:
Can we move this to /test/testutil instead of creating new directory ?

Reply:
The name should also not contain upper case letters.